.eslintrc.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
     'no-fallthrough': warning,
     'require-atomic-updates': 0,
   },
+  globals: {
+    BigInt: true,
+  },
```

Comment:
Do we need to officially drop support for node 8 first?

Reply:
fixed, now globals for test/ directory only

Reply:
Support for Node 8 was dropped already, so this is good to go.